lib/krane/render_task.rb

```diff
@@ -91,6 +91,10 @@ def validate_configuration(template_sets)
       if @filenames.blank?
         errors << "filenames must be set"
       end
+
+      if !@current_sha.nil? && @current_sha.empty?
+        errors << "current-sha is optional but can not be blank"
+      end
```

Comment:
This (and the `@filenames` check right before it) might be better suited to a `TaskConfigValidator` subclass, if we want to follow the same pattern we used with the others tasks; I think it's totally fine to move that in another PR though.

Reply:
Tracking in #637